src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -205,7 +206,7 @@ public class RunResult
     {
         public bool Failed { get; set; }
 
-        public Exception Exception { get; set; }
+        public ExceptionDispatchInfo Exception { get; set; }
```

Comment:
I don't mind, although I don't really see what's wrong with the name, it is essentially still an exception captured in that property?